spirv_glsl.hpp

```diff
@@ -64,6 +65,265 @@ typedef uint32_t AccessChainFlags;
 
 class CompilerGLSL : public Compiler
 {
+protected:
+	struct ShaderSubgroupSupportHelper
```

Comment:
As mentioned earlier, please move this class to the bottom of CompilerGLSL (below protected).
Also, there shouldn't be any inline code for this. The implementation should be done in spirv_glsl.cpp.